lib/utils/url_launcher.dart

```diff
@@ -2,7 +2,7 @@ import 'package:url_launcher/url_launcher_string.dart';
 
 void launchURL(String url) async {
   if (await canLaunchUrlString(url)) {
-    await launchUrlString(url);
+    await launchUrlString(url, mode: LaunchMode.externalApplication);
```

Comment:
By default , keep it launchUrlString(url) without  mode: LaunchMode.externalApplication

Pass an varibale in function `openInExternalApplication` with default value `False`

Do the necessary changes, in the ` mode: LaunchMode.externalApplication` also.

Reply:
i have made the changes in pr#270 , so now only specific links will open with external browser, rest all will open in app web view